### PR TITLE
Address feedback on kernel build instructions

### PR DESCRIPTION
**Pull request by mike-petersen-ni**
Per [AB#2344646](https://dev.azure.com/ni/94b22d7b-ad7b-4f5e-88f0-867910f91c94/_workitems/edit/2344646), we wanted to address some feedback on our instructions for building a kernel.

Should be cherry-picked into all active branches. Will rebase upon request.

### CODE REVIEW

**Review by amstewart — APPROVED** [review]
LGTM. Pending approval from @bstreiff  or @gratian.

File: docs/README.kernel.md [hunk 1]
```diff
@@ -11,6 +11,42 @@ provided by National Instruments may not have the same performance,
...
+ - The "nohz" kernel: a kernel configured to reduce scheduling ticks
+   (configured with `CONFIG_NO_HZ_FULL=y`). This is useful for realtime
+   use cases in certain situations; see the [documentation][nohz-doc]
+   for a thorough description of the altered behavior and its
+   consequences, as well as the NI-provided [README][nohz-readme] that
+   is installed with this kernel.  
+   The packagegroup to install is `packagegroup-ni-nohz-kernel`.
+ - The "debug" kernel: a kernel with debugging information built in.  
+   The packagegroup to install is `packagegroup-ni-debug-kernel`.
+ - The "next" kernel: a kernel based on a newer upstream version, with a
```
Comment (gratian) [inline docs/README.kernel.md:29]: We may want to be a bit more explicit here and say this is an experimental kernel that can be used for testing but it is not recommended for production use cases. We use the "next" kernel to test possible future kernel releases and find issues before they make it into a release. We make no guarantees that a particular version is fully functional, tested, or will be released.

**Review by gratian — APPROVED** [review]
LGTM. Just a minor comment about being explicit that the "next" kernel is not fully tested and may have bugs.

**Review by gratian — APPROVED** [review]

**Comment by mike-petersen-ni** [comment]
@amstewart should this be cherry-picked into other branches?

**Pull request merged**